test/unit/utility/math_test.cpp

```diff
@@ -7,9 +7,10 @@
 
 #include <gtest/gtest.h>
 
+#include <seqan3/core/bit_manipulation.hpp>
 #include <seqan3/utility/math.hpp>
 
-static constexpr size_t max_iterations = 1;//1 << 15;
+static constexpr size_t max_iterations = 1 << 15;
```

Comment:
Sure? How long does the test take now?

Reply:
2^15 = 32768, is not much

Reply:
./utility/math_test  0.56s user 0.00s system 99% cpu 0.563 total

it increased a bit

Reply:
Depends on what you do if 2^15 is much :D